http-server/src/middleware/proxy_request.rs

```diff
@@ -0,0 +1,152 @@
+//! Middleware that proxies requests at a specified URI to internal
```

Comment:
let's rename this to `proxy_get_request` just to be consistent with our struct rename?

Reply:
It makes sense, yep